app/src/pages/SingleContent.vue

```diff
@@ -37,32 +38,38 @@ const isExpiredOrScheduled = computed(() => {
     );
 });
 
-watch(
-    content,
-    async () => {
-        if (!content.value) return;
+watch(content, async () => {
+    if (!content.value) return;
 
-        document.title = isExpiredOrScheduled.value
-            ? `Page not found - ${appName}`
-            : `${content.value.title} - ${appName}`;
+    document.title = isExpiredOrScheduled.value
+        ? `Page not found - ${appName}`
+        : `${content.value.title} - ${appName}`;
```

Comment:
We are now setting the title in two locations in the code, both direcetly using the "document" api as well as through the useHead plugin. We should combine the logic.